docs/tutorials/embedding/OutOfSampleEmbed.ipynb

```diff
@@ -320,12 +405,27 @@
    "outputs": [],
    "source": [
     "# Embed and transform\n",
-    "X_hat, Y_hat = ase.fit_transform(A)\n",
-    "w = ase.transform(a)\n",
+    "X_hat_ase, Y_hat_ase = ase.fit_transform(A)\n",
+    "w_ase = ase.transform(a)\n",
+    "\n",
+    "# Plot\n",
+    "plot_oos(X_hat_ase, w_ase[0], labels, oos_labels=oos_labels, title=\"ASE Out Latent Predictions\")\n",
+    "plot_oos(Y_hat_ase, w_ase[1],labels, oos_labels=oos_labels, title=\"ASE In Latent Predictions\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Embed and transform\n",
+    "X_hat_lse, Y_hat_lse = lse.fit_transform(A)\n",
+    "w_lse = lse.transform(a)\n",
     "\n",
     "# Plot\n",
-    "plot_oos(X_hat, w[0], labels, oos_labels=oos_labels, title=\"Out Latent Predictions\")\n",
-    "plot_oos(Y_hat, w[1],labels, oos_labels=oos_labels, title=\"In Latent Predictions\")"
+    "plot_oos(X_hat_lse, w_lse[0], labels, oos_labels=oos_labels, title=\"LSE Out Latent Predictions\")\n",
+    "plot_oos(Y_hat_lse, w_lse[1],labels, oos_labels=oos_labels, title=\"LSE In Latent Predictions\")"
```

Comment:
Whoa, your weighted plots look way, way off. You have a bug somewhere in the weighted case. Needs to be fixed before we can merge.

<img width="788" alt="Screen Shot 2021-04-28 at 2 11 09 PM" src="https://user-images.githubusercontent.com/12386450/116452357-a5950900-a82b-11eb-9153-e88e0e14ecd0.png">

Reply:
This has been fixed!